dev-lang/golang/golang-1.4.3.recipe

```diff
@@ -0,0 +1,73 @@
+SUMMARY="The Go Programming Language"
+DESCRIPTION="The Go Programming Language is an open source project to make \
+programmers more productive.
+
+Go is expressive, concise, clean, and efficient. Its concurrency \
+mechanisms make it easy to write programs that get the most out of multicore \
+and networked machines, while its novel type system enables flexible and \
+modular program construction. Go compiles quickly to machine code yet has the \
+convenience of garbage collection and the power of run-time reflection. It's a \
+fast, statically typed, compiled language that feels like a dynamically typed, \
+interpreted language."
+HOMEPAGE="https://golang.org/go"
+COPYRIGHT="2009-2018 The Go Authors"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://dl.google.com/go/go$portVersion.src.tar.gz"
+CHECKSUM_SHA256="9947fc705b0b841b5938c48b22dc33e9647ec0752bae66e50278df4f23f64959"
+SOURCE_DIR="go"
+PATCHES="golang-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 !x86 ?x86_64"
+SECONDARY_ARCHITECTURES="!x86"
+
+PROVIDES="
+	golang$secondaryArchSuffix = $portVersion
+	cmd:go$secondaryArchSuffix = $portVersion
+	cmd:gofmt$secondaryArchSuffix = $portVersion
+	cmd:godoc$secondaryArchSuffix = $portVersion
+	"
+if [ -n "$secondaryArchSuffix" ]; then
+PROVIDES="$PROVIDES
+	cmd:go = $portVersion
+	cmd:gofmt = $portVersion
+	cmd:godoc = $portVersion
+	"
+fi
+
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:git
+	"
+
+BUILD()
+{
+	echo "$portVersion" > VERSION
+	export GOROOT_FINAL=/system/$relativeDevelopLibDir/go
+	cd src
+	DISABLE_ASLR=1 ./make.bash
+	cd ..
+	export GOPATH=/tmp
+	export GOROOT="$sourceDir"
+}
+
+INSTALL()
+{
+	mkdir -p $developLibDir/go/
+	cp -r src pkg doc $developLibDir/go/
+	mkdir -p $binDir
+	cp bin/* $binDir/
```

Comment:
```diff
 INSTALL()
 {
-	mkdir -p $developLibDir/go/
+	mkdir -p "$developLibDir"/go/
-	cp -r src pkg doc $developLibDir/go/
+	cp -r src pkg doc "$developLibDir"/go/
-	mkdir -p $binDir
+	mkdir -p "$commandBinDinDir"
-	cp bin/* $binDir/
+	cp bin/* "$commandBinDinDir"
 }
```
(You can discard the cosmetic changes about the quotes if you wish, but I recommend using them in new recipes.)

Reply:
Same comment as above.
I guess I should check what we have for the existing recipe (1.3) of golong on x86_64 and golang_x86 on x86_gcc2.